templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html.twig' %}
+
+{% block main %}
+
+    <ol class="breadcrumb">
+        <li><a href="{{ path('home') }}">Home</a></li>
+        <li class="active">{{ msg('error-breadcrumb', [exception.statusCode]) }}</li>
+    </ol>
+
+    <div class="error alert alert-danger">
+        <div class="icon">
+            <span class="glyphicon glyphicon-remove-circle"></span>
+        </div>
+       <div class="message">
+            <p>
+                {{ msg('error-page-header') }}
+            </p>
+            <p>
+                <strong>{{ msg('error-page-details', [exception.message]) }}</strong>
+            </p>
+            <p>
+                {% set issue_link %}{#
+                    #}<a href="https://phabricator.wikimedia.org/maniphest/task/edit/form/1/?projects=ws-export">{#
+                        #}{{ msg( 'error-page-issue-link' ) }}{#
+                    #}</a>{#
+                #}{% endset %}
```

Comment:
Not sure if it works here but I think you can [remove the whitespace](https://symfony.com/blog/better-white-space-control-in-twig-templates) with:
```
{%- set issue_link %}
    <a href="https://phabricator.wikimedia.org/maniphest/task/edit/form/1/?projects=ws-export">
        {{- msg( 'error-page-issue-link' ) -}}
    </a>
{% endset -%}
```

Reply:
Oh, that's a cool trick! Thanks. Done. It's only required at the end, to stop the period floating out on its own.